### PR TITLE
Add's line breaks to comment text

### LINKED ISSUE #2082: Fix breaks in Lab comments and narratives
## Action Requested

Lab comments and narratives currently do not maintain the original breaks/newlines, making it more difficult to read.
As a user, I want to see the breaks that separate the text so that it’s easier to read.

## Acceptance Criteria

GIVEN an eCR with lab info
WHEN a lab result has a narrative and/or comment(s)
THEN the viewer should keep the original spacing/new lines of the text.

What should be changed:
- Lab Info → Components table → Lab Comments (if available)
- Lab Info → Narrative (if available)

## Testing

Test fixtures, snapshot, and other unit tests may need to be updated.

## Designs

[Figma designs](https://www.figma.com/file/tXWUeV3Kjehd7VNqHojZYq/eCR-Viewer?type=design&node-id=871-2283&mode=design&t=LpQrioiHotZiGBxo-0)

**NOTE From Sarah**: Even if the original has too many breaks between text, we would rather have all the breaks than none at all.

## Background & Context

Examples below from `61f13d32-b25c-4fb6-998b-73de2dbf87f2` ([Link](http://localhost:3000/view-data?id=1.2.840.114350.1.13.478.3.7.8.688883.230886#treatment-details) to eCR viewer)

**Example: Lab Comments (see: Hepatitis C IgG w/Rfx PCR)**

**Example: Lab Narrative (see: FISH ALL PnL)**

### PR DESCRIPTION
**Pull request by nickbristow**
# PULL REQUEST
Adds line breaks to comment text

## Summary
When a piece of text has `<br>` separators, it adds a paragraph tag around each piece.

## Related Issue
Fixes #2082 

## Additional Information
Anything else the review team should know?

## Checklist

- [ ] If this code affects the other scrum team, have they been notified? (In Slack, as reviewers, etc.)

[//]: # (PR title: Remember to name your PR descriptively!)

### CODE REVIEW

**Comment by sarahtress** [comment]
@nickbristow could you add some screenshots so I can review? thanks!

**Comment by nickbristow** [comment]
@sarahtress So we don't have a ton of data outside the HTML version, but I can see that they are separate strings and they are using `<br>` elements. So i hope this is good.
DIBBs_eCR_Viewer

**Comment by sarahtress** [comment]
this looks right to me!

File: containers/fhir-converter/CustomFhir/CustomFilters.cs [hunk 1]
```diff
@@ -177,9 +238,13 @@ public static string ToHtmlString(object data)
       }
       else if (data is IList listData)
       {
-        foreach (var row in listData)
+        for (int i = 0; i < listData.Count; i++)
         {
-          stringBuilder.Append(ToHtmlString(row));
+            stringBuilder.Append(ToHtmlString(listData[i]));
```
Comment (BobanL) [inline containers/fhir-converter/CustomFhir/CustomFilters.cs:243]: Can we try to resolve this warning?

Comment (nickbristow) [reply containers/fhir-converter/CustomFhir/CustomFilters.cs]: Should be good now

**Review by BobanL — COMMENTED** [review]
👯

**Review by BobanL — APPROVED** [review]

**Pull request merged**